test/Microsoft.ML.Core.Tests/UnitTests/TestEntryPoints.cs

```diff
@@ -3846,7 +3858,11 @@ public void EntryPointChainedTrainTestMacros()
             validateAuc(metrics);
         }
 
-        [RetryFact]
+#if INNER_LOOP
+        [Fact(Skip = "skip due to flaky")]
```

Comment:
Can you please change the message to something like "Skipping test temporarily. This test will be re-enabled once the cause of failures has been determined"